connect.go

```diff
@@ -220,10 +226,73 @@ func (cli *Client) ParseConnect(ctx context.Context, deviceID string, targets []
 	return cli.Connect(ctx, deviceID, parsedTargets)
 }
 
-// Connect establishes a connection to a remote device.
-func (cli *Client) Connect(ctx context.Context, deviceID string, targets []RemoteAccessTarget) error {
+// connect establishes a connection to a remote device.
+func (cli *Client) connect(ctx context.Context, deviceUUID, edgeHost string, targets []RemoteAccessTarget) error {
+	edgeURL := fmt.Sprintf("https://%s/device/%s", edgeHost, deviceUUID)
+
+	var tlsConfig *tls.Config
+
+	// for testing purposes, allow connections to localhost without verifying the certificate
+	if strings.HasPrefix(edgeHost, "edge:") || strings.HasPrefix(edgeHost, "localhost:") {
+		tlsConfig = &tls.Config{
+			InsecureSkipVerify: true,
+		}
+	}
+
+	client, err := transport.NewClient(ctx, edgeURL, cli.authToken, tlsConfig)
+	if err != nil {
+		return fmt.Errorf("error initializing remote access client: %w", err)
+	}
+
+	// close the client and all local listeners when the context is cancelled
+	closers := []io.Closer{client}
+	defer func() {
+		for _, closer := range closers {
+			_ = closer.Close()
+		}
+	}()
+
+	for _, target := range targets {
+		localHostPort := fmt.Sprintf("localhost:%s", target.LocalPort)
```

Comment:
Will this also support `stdio`as localport (for integration with ssh)?
https://qbee.io/docs/qbee-cli-ssh-integration.html

Reply:
I was kind of thinking about adding dedicated command `qbee-cli shell <deviceID>` to do that directly into an authenticated shell, but I can definitely add support for it in the `connect` command to retain compatibility with the old agents.

Reply:
Done.